app/Plot.php

```diff
@@ -24,7 +24,8 @@ class Plot extends Model
         'canopy',
         'subcanopy',
         'ground_cover',
-        'recorders'
+        'recorders',
+        'last_measured_at',
     ];
 
```

Comment:
We need to auto cast so that we can do comparison operations.
```suggestion
    protected $casts = [
        'last_measured_at' => 'date',
    ];
```